### ISSUE TITLE
Recent Transactions display issue

### ISSUE THREAD

**Issue by misterdna** [bitcoin/bitcoin#6615]
I'm a GitHub noob and not a technical person, so I apologize if this has been dealt with, or I'm not presenting this right.

I'm running Core v0.11.0 (64-bit) on Mac.

In the overview screen, there's a display issue where the top transaction label text is cut off (the letters have about 2/3 cut off the top). It's as if that line of text is too high, too close to the date/time of the transaction.

**Comment by jonasschnelli** [comment]
@misterdna: would it be possible for you to post a screenshots for this issue?

**Comment by misterdna** [comment]
The top transaction that was having the display issue exposed a bit about my privacy (as all my transactions are labeled), so I just did a new transaction, assuming it would now have the display issue (as I assumed it was an issue only with the newest transaction). However, the line/transaction having the issue simply moved down one on the list.

I then realized I could change the label on the transaction in question (to protect my privacy), so that's what I did.

Bottom line, the display issue seemed to stick with the 8/15/15 transaction, it didn't stay at the top. Please let me know if you need more info.

![screen shot 2015-09-02 at 5 34 03 am](https://cloud.githubusercontent.com/assets/4542126/9631243/7096030c-5134-11e5-8d4f-b0b44c759a9d.png)

**Comment by jonasschnelli** [comment]
Can't reproduce this issue. Can you give me some more details about your setup? Do you use the official  binaries? What OSX version do you use?

Looks good here OSX:
<img width="403" alt="bildschirmfoto 2015-11-13 um 16 56 21" src="https://cloud.githubusercontent.com/assets/178464/11150275/99999ec2-8a27-11e5-914d-17759bcaa3ee.png">

Windows:
<img width="395" alt="bildschirmfoto 2015-11-13 um 16 59 54" src="https://cloud.githubusercontent.com/assets/178464/11150380/15555088-8a28-11e5-90ab-dea5a4cc7825.png">

Linux:
<img width="212" alt="bildschirmfoto 2015-11-13 um 17 01 58" src="https://cloud.githubusercontent.com/assets/178464/11150429/4ace3216-8a28-11e5-8121-e0e808cc754f.png">

**Comment by misterdna** [comment]
I downloaded from Bitcoin.org, running 10.10.5.

Note this turned out to be an extremely isolated incident. Turned out the misaligned text stuck with the transaction, slowing moving through the Overview screen. Now that the transaction has cleared past that screen, there’s been no repeating of the issue. Probably not worth perusing. When I first reported it, I had assumed it was a pervasive issue with the Overview screen.

Thanks for following up!

**Comment by jonasschnelli** [comment]
Thanks for the report. Closing then.
